### PR TITLE
Reader guide improvements

### PR DESCRIPTION
**Pull request by bbtora**
Let me know of any other changes.

Optional: Provide some illustrations for landscape scaling to be added in this PR.

### CODE REVIEW

**Comment by bbtora** [comment]
Sorry for the badge positioning mistake.

Page Transitions has the wrong description so the newly created badge is useless.

File: src/help/guides/reader.md [hunk 1]
```diff
@@ -142,14 +140,14 @@ This allows you to control what color the background should be in reader. Backgr
     <figure class="centered">
 	    <img height="400" intrinsicsize="208x500" width="260" :src="$withBase('/assets/media/backgroundcolorblack.png')">
         <h4>Black</h4>
-        <p>Surrounds page with black. With a Amoled screen you can save some battery by using this to read some more series.</p>
+        <p>Surrounds page with black. With an Amoled screen you can save battery by using this.</p>
```
Comment (Soitora) [inline src/help/guides/reader.md]: Maybe uppercase `AMOLED`?

File: src/help/guides/reader.md [hunk 2]
```diff
@@ -181,25 +180,25 @@ This settings shows the page number at the bottom of the page in current page/to
 This setting decodes images in ARGB888 format to allow the reader to display more colors.
 
 ::: warning
-This setting will only show up on Oreo or later
+This setting will only show up on smartphones running Android Oreo and later versions.
```
Comment (Soitora) [inline src/help/guides/reader.md]: Maybve `Android 8.0 Oreo`?

File: src/help/guides/reader.md (hunk 1 above)
Comment (bbtora) [reply src/help/guides/reader.md]: Fixed in latest commit.

File: src/help/guides/reader.md (hunk 2 above)
Comment (bbtora) [reply src/help/guides/reader.md]: Fixed in latest commit,

**Pull request merged**